notebook-lm-clone/src/memory/memory_layer.py

```
@@ -0,0 +1,326 @@
+                logger.info(f"Created new session: {self.session_id}")
+            else:
+                # Try to use existing session, create if doesn't exist
+                try:
+                    self.zep_client.thread.get(self.session_id)
+                    logger.info(f"Using existing session: {self.session_id}")
+                except:
+                    self.zep_client.thread.create(thread_id=self.session_id, user_id=self.user_id)
+                    logger.info(f"Created session: {self.session_id}")
+                    
+        except Exception as e:
+            logger.error(f"Error setting up user/session: {str(e)}")
```

Ruff (0.13.1)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)